src/components/application_manager/src/commands/mobile/subscribe_button_request.cc

```diff
@@ -93,7 +93,8 @@ void SubscribeButtonRequest::Run() {
 bool SubscribeButtonRequest::IsSubscriptionAllowed(
     ApplicationSharedPtr app, mobile_apis::ButtonName::eType btn_id) {
   if (!app->is_media_application() &&
-      ((mobile_apis::ButtonName::SEEKLEFT == btn_id) ||
+      ((mobile_apis::ButtonName::PLAY_PAUSE == btn_id) ||
```

Comment:
It looks like there is already a test for a non-media application attempting to subscribe to a media button [here](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/application_manager/test/commands/mobile/subscribe_button_request_test.cc#L89-104), but it only tests the SEEK_LEFT button.

Should I expand this to test all media buttons, or is it fine as is?